Governance.md

```diff
@@ -0,0 +1,78 @@
+
+## Goals of this document
+
+Clarify:
+0. values of the project.
+1. how to get a PR merged
+2. how long you should need to wait for a review
+3. how to resolve conflicts in project direction
+4. how to make larger and disruptive changes
+
+## Values of rules_scala
+
+1. Correctness: bazel values fully reproducible builds which are reliable and hermetic (only depend on the current state of your repository)
+2. Speed: we want to build as fast as possible. In a large repository, we believe caching and remote parallel builds are the only way to achieve this. Without correctness, we can't cache and use remote builds, so correctness enables speed. We will almost never trade off correctness for speed.
+3. Usability: we want it to be as easy as possible to adopt and to maintain a build with these rules. Usability is a lower priority than correctness or speed. We will virtually never trade either correctness or speed for more usability. Usability includes stability: we seek to absolutely minimize breaking changes. We will prefer to keep some warts or deprecated methods than force users to constantly make changes to keep their builds on the latest version of the rules.
+
+## How to make a PR
+We welcome pull-requests and do all development in the open. Pull requests
+should be in the smallest testable units. Ideally a PR is less than 400 lines of
+change. In rare cases it will be more. If you PR is much larger than that, we
+will likely ask you to split it into smaller changes. You should see a comment
+in 24 business hours on your PR. Your PR *may* be ignored if it has a red CI. It
+is your responsibility to get your PR green, or to ask for clarification if you
+doubt a test failure is due to your change.
+
+Code review increases the quality of our code.  We welcome code review by anyone:
+you don't need to be a maintainer to offer your review.
+
+PRs are merged when 2 or more maintainers have approved the PR and there are no
+blocking objections by any maintainers, and not before 24 business hours have
+passed to allow sufficient time review. When there are disagreements by
+maintainers that cannot be settled in PR discussion, we will use a video chat
+scheduled when 3/4 or more of the maintainers can make it. After the video chat we
+will choose a course of action. If we cannot all agree, if 2/3 or more support
+merging a change, it will be done, otherwise we bias to no-merge.
+
+## Reporting bugs
+We welcome bug reports but keep in mind this is a community driven OSS project.
+There is no one currently paid to try to fix bugs that have been filed. The
+contributors and maintainers are happy to help coach new contributors to fix
+bugs. In most cases, bug fixes should be simple since these rules have been used
+in production at multiple companies for several years now.
+
+An ideal bug report will be a PR that adds a failing test to the repo. This
+makes it easiest for another contributor to try to make that test green.
+
+## Proposing designs
```

Comment:
Before we merge, I would like to elaborate on the design process a bit more. I think Bazel has a pretty great system going on with https://bazel.build/designs/ Doing something similar here would be great. I don't want to make it too cumbersome, but it would be nice to have an index of proposals with their date, reviewers, status. Perhaps we can still get all of that with just PRs, or perhaps checked in MD documents or Google Docs are more appropriate.

Reply:
I personally prefer checked in MD docs so the history is all in one repo. That was my intent. Would be interested to hear other's opinions.

Reply:
I like the idea of having it all as part of the repo. There could be an index md file as well if that helps organize things.

Reply:
I dig it. Sounds like we agree on a checked in MD with an index. :+1: